### PR TITLE
added animation

### PR DESCRIPTION
**Pull request by ghost**
adding shadow animations to the whole of the card.

### CODE REVIEW

**Comment by Aminfrka** [comment]
can you explain further what you are trying to achieve

**Comment by ghost** [comment]
this animation is meant to shadow the text as soon as the dom content is loaded.

File: app.css [hunk 1]
```diff
@@ -100,7 +100,8 @@ img {
   color: rgb(219, 121, 121);
 }
 
-.active .profile-img::after {
+.active .profile-img::after,
```
Comment (Aminfrka) [inline app.css:103]: give more explaination on this, please.

Comment (Aminfrka) [reply app.css:103]: is this necessary

**Pull request merged**